packages/pds/src/db/migrations/20221021T162202001Z-init.ts

```diff
@@ -32,6 +33,11 @@ export async function up(db: Kysely<unknown>, dialect: Dialect): Promise<void> {
       if (!err?.detail?.includes?.('(pg_trgm) already exists')) throw err
     }
   }
+
+  // Postgres uses the type `bytea` for variable length bytes
+  // Kysely doesn't have this as a "supported data type", but it is 😛
+  const binaryDatatype = dialect === 'sqlite' ? 'blob' : ('bytea' as any)
```

Comment:
Small peas, but if you want to avoid the `any`, I think this is how kysely supports types it may not know about:
```suggestion
  const binaryDatatype = dialect === 'sqlite' ? 'blob' : sql`bytea`
```

Reply:
ah excellent 🙏